src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -671,9 +674,14 @@ def image_mask_iterator():
             ],
             id=roi_ids,
         )
-
-        image_segmentation.create_plane_segmentation(**plane_segmentation_kwargs)
-
+        plane_segmentation = PlaneSegmentation(**plane_segmentation_kwargs)
+        if include_roi_centroids:
+            tranpose_roi_location_axes = (1, 0) if len(segmentation_extractor.get_image_size()) == 2 else (1, 0, 2)
```

Comment:
Can you write a brief comment on which case each of this if covers. Or maybe an appropriate name for the boolean variable instead? `dealing_with_non_volumetric_data = len(segmentation_extractor.get_image_size())==2` or something like this?

I just think that it took me a minute or two what this was for and I think I will forget in the future.

Reply:
Added comment